django/core/management/commands/loaddata.py

```diff
@@ -292,6 +301,12 @@ def parse_name(self, fixture_name):
 
         return name, ser_fmt, cmp_fmt
 
+    def should_skip(self, object):
+        """
+        Tests if the specified object should not be saved to the database.
+        """
+        return (object._meta.app_config in self.excluded_apps) or (object.__class__ in self.excluded_models)
```

Comment:
I couldn't find any old-style class usage relevant to this in Django codebase so we probably could use `type(object)` instead of `object.__class__`.